core/src/test/java/net/sourceforge/jnlp/DefaultLaunchHandlerTest.java

```diff
@@ -33,113 +33,87 @@
 
 package net.sourceforge.jnlp;
 
-import net.adoptopenjdk.icedteaweb.testing.ServerAccess;
 import net.adoptopenjdk.icedteaweb.xmlparser.ParseException;
-import net.sourceforge.jnlp.util.logging.LogConfig;
+import net.sourceforge.jnlp.util.logging.NoStdOutErrTest;
 import net.sourceforge.jnlp.util.logging.OutputController;
+import net.sourceforge.jnlp.util.logging.StdInOutErrController;
+import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Test;
 
 import java.io.ByteArrayOutputStream;
-import java.io.PrintStream;
-import java.lang.reflect.Field;
 
+import static java.nio.charset.StandardCharsets.UTF_8;
 import static net.sourceforge.jnlp.LaunchException.SEVERE;
 import static org.junit.Assert.assertEquals;
 
-public class DefaultLaunchHandlerTest {
+public class DefaultLaunchHandlerTest extends NoStdOutErrTest {
 
-    private static class LocalLogger extends OutputController {
+    private final OutputController l = OutputController.getLogger();
 
-        private static class AccessibleStream extends PrintStream {
+    private ByteArrayOutputStream out;
+    private ByteArrayOutputStream err;
 
-            AccessibleStream(ByteArrayOutputStream out) {
-                super(out);
-            }
-
-            public ByteArrayOutputStream getOut() {
-                return (ByteArrayOutputStream) out;
-            }
-        }
-
-        LocalLogger() {
-            super(new AccessibleStream(new ByteArrayOutputStream()), new AccessibleStream(new ByteArrayOutputStream()));
-            try{
-                Field f = LogConfig.class.getDeclaredField("logToStreams");
-                f.setAccessible(true);
-                f.set(LogConfig.getLogConfig(), true);
-            }catch(Exception ex){
-                ServerAccess.logException(ex);
-            }
-        }
-
-        String getStream1() {
-            return ((AccessibleStream) (super.getOut())).getOut().toString();
-        }
-
-        String getStream2() {
-            return ((AccessibleStream) (super.getErr())).getOut().toString();
-        }
+    @Before
```

Comment:
I ask myself if we should have an @After to reset the behaviour for tests that will be executed after the tests of this class